src/prefect/settings.py

```diff
@@ -562,6 +562,19 @@ def default_cloud_ui_url(settings, value):
 )
 """Whether to style console logs with color."""
 
+PREFECT_LOGGING_MARKUP = Setting(
```

Comment:
I think we often use `_ENABLED` for settings like this, but I see you didn't for the logging colors option so I don't feel strongly here.